docs/resources/gno-interrealm.md

```diff
@@ -320,14 +320,14 @@ func AnotherPublic(cur realm) {
 ### MsgRun
 
```

Comment:
The intro of this doc explains the differences between P and R but does not mention E. A reader would probably be confused by the example below without any brief preliminary explanation.